config/ota-contrib-declarations-production.json

```diff
@@ -1,12 +1,12 @@
 {
   "services": {
-    "declarationsPath": "../ota-services-all/declarations"
+    "declarationsPath": "../declarations/declarations"
```

Comment:
There is duplications between instance production config files.
I suggest to create a `production.json` config file with the factorized configs keys shared by all instances and rename `ota-contrib-declarations-production` to `contrib` (and so on…) with only the overridden keys.

Reply:
Done